### PR TITLE
Fix Patrol Grammar Error

### PR DESCRIPTION
**Pull request by Clumxy**
## About The Pull Request

This fixes the patrol error in fst_med_greenleaf_gatheringelder_lorehealingstorylocked1.  ignore the windows.py thing, ill remove the extra line. 

## Linked Issues

https://github.com/ClanGenOfficial/clangen/issues/2849

## Proof of Testing

image

## Changelog/Credits

patrol fixes?

### CODE REVIEW

File: scripts/game_structure/windows.py [hunk 1]
```diff
@@ -2233,7 +2233,8 @@ def __init__(self, source_screen: "Screens"):
             },
         )
         self.text_block.rebuild_from_changed_theme_data()
-
+    
+
```
Comment (archanyhm) [inline scripts/game_structure/windows.py]: Why are these added? I'd advise removing them. Changes to unrelated files should be avoided.

Comment (Clumxy) [reply scripts/game_structure/windows.py]: will remove them!

**Comment by Clumxy** [comment]
@archanyhm fixed!

**Comment by Clumxy** [comment]
bumping!

**Comment by scribblecrumb** [comment]
Looking at this bug, it doesn't seem to be an issue with how the patrol is written, rather it's an issue with the backend patrol code.  Your fix would just be a temporary stopgap rather than a true fix.  I've corrected the underlying issue in #2862 so I will be closing this PR.  Also, just a small sidenote, you don't need to worry about "bumping" PRs.  Seniors will get to reviewing it when we're able to.  And if you ever wish to request a review, you can do so via the builtin github "request review" feature.
